lib/action_cable/channel/broadcasting.rb

```diff
@@ -0,0 +1,25 @@
+module ActionCable
+  module Channel
+    module Broadcasting
+
+      # Broadcast a hash directly to a named <tt>broadcasting</tt>. It'll automatically be JSON encoded.
+      def broadcast(broadcasting, message)
+        ActionCable.server.broadcast(broadcasting, message)
+      end
+
+      # Broadcast a hash a unique broadcasting for this <tt>object</tt> in this channel.
+      def broadcast_to(object, message)
+        broadcast(broadcasting_for([channel_name, object]), message)
+      end
+
+      # Generates a unique named broadcasting for the <tt>object</th>.
+      def broadcasting_for(object)
+        case
+        when object.is_a?(Array)               then object.map{ |o| broadcasting_for(o) }.join(':')
+        when object.respond_to?(:to_global_id) then object.to_global_id.to_param
+        else                                        object.to_param
+        end.to_s
```

Comment:
Doesn't `to_param` return a string already? `join` definitely does, so I'd rather :scissors: it.